<launch>
	

	<param name="device" type="str" value="/dev/ttylightware_altitude" />

	
		<node   pkg="altitude_sensor" 
			name="altitude_sensor" 
			type="altitude_sensor_node" />

		<param name="serial_port" type="string" value="/dev/lightware_altitude" />
	
	
	

</launch>


